src/main/kotlin/net/lumalyte/lg/interaction/menus/guild/GuildStatisticsMenu.kt

```
@@ -1113,195 +1086,342 @@ class GuildStatisticsMenu(private val menuNavigator: MenuNavigator, private val
+            player.sendMessage(lang.msg("menu.statistics.export.header", "guild" to guild.name))
+            player.sendMessage(lang.msg("menu.statistics.export.level", "level" to guild.level))
+            player.sendMessage(lang.msg("menu.statistics.export.kills", "kills" to killStats.totalKills, "deaths" to killStats.totalDeaths))
+            player.sendMessage(lang.msg("menu.statistics.export.kdr", "ratio" to decimalFormat.format(killStats.killDeathRatio)))
+            player.sendMessage(lang.msg("menu.statistics.export.wars", "wins" to warsWon, "losses" to warsLost))
+            player.sendMessage(lang.msg("menu.statistics.export.balance", "amount" to decimalFormat.format(balance)))
+            player.sendMessage(lang.msg("menu.statistics.export.members", "count" to memberCount))
+            if (topKillerNames.isNotEmpty()) {
+                player.sendMessage(lang.msg("menu.statistics.export.top_killers", "killers" to topKillerNames))
+            }
+            player.sendMessage(lang.msg("menu.statistics.export.footer"))
         } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)